### PR TITLE
JAMES-2980 Two James with ssl remote delivery test

### PR DESCRIPTION
**Pull request by trantienduchn**
testing the remote delivery of the implementation in https://github.com/linagora/james-project/pull/2892

### CODE REVIEW

**Comment by Arsnael** [comment]
Read it

File: dockerfiles/run/guice/cassandra/destination/conf2/dnsservice.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  KIND, either express or implied.  See the License for the    
+  specific language governing permissions and limitations      
+  under the License.                                           
+ -->
+
+<!-- Read https://james.apache.org/server/config-dnsservice.html for further details -->
+
+<dnsservice>
+  <servers>
+    <server>8.8.8.8</server>
```
Comment (remk) [inline dockerfiles/run/guice/cassandra/destination/conf2/dnsservice.xml]: Do we want to use google dns ? :trollface:

Comment (Arsnael) [reply dockerfiles/run/guice/cassandra/destination/conf2/dnsservice.xml]: Maybe not..? I noticed the other day though that all our `dnsservice.xml` have that dns declared :trollface:

Comment (trantienduchn) [reply dockerfiles/run/guice/cassandra/destination/conf2/dnsservice.xml]: I didn't know it was there :) I had a hard time makes two dockers james can recognize each other's domain through DNS

**Comment by trantienduchn** [comment]
It works, but having some error logs due to the incorrect SSL setup.

**Pull request closed without merging**